src/fontra/views/editor/scene-controller.js

```diff
@@ -637,10 +637,28 @@ export class SceneController {
       parseSelection(relevantSelection);
     this.contextMenuState.pointSelection = pointSelection;
     this.contextMenuState.componentSelection = componentSelection;
+
+    // relevantSelection does not include cmd + A selection,
```

Comment:
There is a bit of misunderstanding here: the rest of the context menus only get enable of you right-click _on_ part of the selection or on something selectable. If you click anywhere else, the context menu items will not be enabled.

So it's not that "relevantSelection does not include cmd-A selection", it totally does, as long as you right-click on something that is selected or is selectable.

This is by design. We could argue about that design, but that's how all other context menu items here behave, so this one should also behave like that.

This means you can get rid of the additional parseSelection call, and just use pointSelection instead of pointIndices.

Reply:
Thanks for the explanation. I was not aware of that. I changed it like suggested.